internal/cli/internal/command/services/postgres/util.go

```diff
@@ -0,0 +1,46 @@
+package postgres
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"time"
+
+	"github.com/superfly/flyctl/internal/client"
+)
+
+func WaitForMachineState(ctx context.Context, client *client.Client, appId, machineId, state string) error {
```

Comment:
Oops, I spotted another thing. In Go we typically use [MixedCaps](https://tip.golang.org/doc/effective_go#mixed-caps) so unfortunately `machineId` should be `machineID`.

Reply:
haha, all good.  I'll update.